### PR TITLE
Feature/unit tests

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Re-enable unit tests and add pre-push verification.

#### Screenshots or example usage
screen shot 2018-04-19 at 16 41 33

#### Note
Some tests are disabled `xit`. That is because they were copied from the old repo and their adjacent feature has not yet been implemented in the Render version of the app (PR at the old repo).

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/reducers/Invoices/index.spec.js [hunk 1]
```diff
@@ -24,21 +24,20 @@ const initialState = {
...
     fetchErrors: 0,
   },
-  cancelResults: {},
   uploadInvoices: {},
   hasInvoices: false,
 }
 
 describe('invoiceReducer', () => {
-  it('should return the initial state', () => {
+  xit('should return the initial state', () => {
```
Comment (gabrielgodoy) [inline react/reducers/Invoices/index.spec.js]: Commented test case

File: react/reducers/Invoices/index.spec.js [hunk 2]
```diff
@@ -310,48 +309,59 @@ describe('invoiceReducer', () => {
     })
   })
 
-  it('should handle REQUEST_CANCEL_INVOICE', () => {
+  xit('should handle REQUEST_CANCEL_INVOICE', () => {
```
Comment (gabrielgodoy) [inline react/reducers/Invoices/index.spec.js]: commented test case

File: react/reducers/Invoices/index.spec.js [hunk 3]
```diff
@@ -310,48 +309,59 @@ describe('invoiceReducer', () => {
...
+      invoiceUpdate: {
+        success: false,
+        error: false,
+        code: null,
+      },
+    })
   })
 
-  it('should handle RECEIVE_CANCEL_INVOICE', () => {
+  xit('should handle RECEIVE_CANCEL_INVOICE', () => {
```
Comment (gabrielgodoy) [inline react/reducers/Invoices/index.spec.js]: commented test case

File: react/reducers/Invoices/index.spec.js [hunk 4]
```diff
@@ -310,48 +309,59 @@ describe('invoiceReducer', () => {
...
+        status: 'Cancelled',
+      },
+      invoiceUpdate: {
+        success: true,
       },
     })
   })
 
-  it('should handle RECEIVE_CANCEL_INVOICE_ERROR', () => {
+  xit('should handle RECEIVE_CANCEL_INVOICE_ERROR', () => {
```
Comment (gabrielgodoy) [inline react/reducers/Invoices/index.spec.js]: commented test case

File: react/reducers/Invoices/index.spec.js [hunk 5]
```diff
@@ -310,48 +309,59 @@ describe('invoiceReducer', () => {
...
+      isLoading: false,
+      invoiceUpdate: {
+        error: true,
+        code: 2003,
       },
     })
   })
 
-  it('should handle REQUEST_INVOICE', () => {
+  xit('should handle REQUEST_INVOICE', () => {
```
Comment (gabrielgodoy) [inline react/reducers/Invoices/index.spec.js]: commented test case

File: react/reducers/Invoices/index.spec.js (hunk 1 above)
Comment (kevinch) [reply react/reducers/Invoices/index.spec.js]: Yes, i wrote a _note_ in the description of the PR above about that.

**Comment by gabrielgodoy** [comment]
LGTM

**Pull request merged**